src/main/java/thirdtask/MagicSquare.java

```diff
@@ -0,0 +1,43 @@
+package thirdtask;
+
+import utils.*;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class MagicSquare {
+    private static final String INPUTFILE = "src/main/resources/input_files/input_thirdtask";
+    private static final String OUTPUTFILE = "target/logs/output_thirdtask.txt";
+
+    public static void main(String[] args) throws IOException {
+        int[][] square = InputOutputUtils.readTwoDimensionalArrayFromFile(INPUTFILE);
+        if (checkIfMagicSquare(square)) {
+            System.out.println("This is a magic square");
+            InputOutputUtils.writeTwoDimensionalArrayToFile(square, OUTPUTFILE);
+        } else {
+            System.out.println("This is NOT a magic square");
```

Comment:
please handle the negative message in the output file

Reply:
Made some changes.